apps/storybook/.storybook/main.js

```diff
@@ -32,7 +34,7 @@ export default {
     '@storybook/addon-mdx-gfm',
   ],
   features: {
-    storyStoreV7: false,
+    storyStoreV7: { isWindows },
```

Comment:
think you meant this:
```suggestion
    storyStoreV7: isWindows,
```